styles/components/_filter-toggle.scss

```diff
@@ -14,19 +21,22 @@
   }
 
   &__text {
+    width: 100%;
     border: 1px solid color("white");
-    display: inline-block;
+    display: flex;
+    align-items: center;
+    justify-content: center;
     font-size: font-size("xs");
-    height: 3.125rem;
-    line-height: 3.125;
+    padding: space("sm");
     text-align: center;
     text-transform: capitalize;
     transition: background-color 250ms, color 250ms ease-in-out;
```

Comment:
I know that this isn't new code, and this suggestion doesn't have to be done in this PR, but it can be nice to create a variable for transition times. So you could have `$transition-quick: 250ms` and that way you only have to change it in one place if the value ever needs adjusted.

Reply:
Good idea! And even though it's not explicitly apart of this PR I think it makes sense for me to go ahead and include it here.